src/Fixtures/index.js

```diff
@@ -53,18 +53,22 @@ export const EquipmentLabels = {
   FULL: 'Full equipment',
 };
 
+export const Sources = {
```

Comment:
`VideoSources` ?

Reply:
Hmm, don't know if it is better as it is between `animation`, `youtube` and `vimeo`... `sourceType` ?

Reply:
It's just that `Source` is so generic I would exect it to be the source of the drill, like `UltimateApp`, `RiseUp`, `Other` (close to "author")

Suggestions : `DisplayType`, `IllustrationType` ?

Reply:
Ah, yes, agree with you
Let's go for `IllustrationType`

Reply:
I choose `IllustrationType`
By the way, do you think we change the naming in steps (`sourceType` and `source`) or no ?
And I renamed `source` in `drill` by `author`

Reply:
>  By the way, do you think we change the naming in steps (sourceType and source) or no ?

Probalby yes, to be consistent?